### PR TITLE
Bezhan oss ccp integration

### PR DESCRIPTION
**Pull request by bezhan-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

**Comment by bezhan-msft** [comment]
For custom connectors - do they submit straight to master? @mamurshe

File: README.md [hunk 1]
```diff
@@ -1,34 +1,109 @@
...
+For new custom connectors, create a directory under the ```custom-connectors``` directory and place the connector files in the sub-folder. Ensure that a clear, structured, readme.md is included. 
+
+#### Certified Connectors
+
+Updates to certified connectors must first be made through a pull request to the ```dev``` branch. 
+Once in the ```dev``` branch, the certified connector owner can submit the connector for certification and deployment throughout the Power Platform. 
+Once certified, the Microsoft Certification team will handle merging the updates from ```dev``` to ```master```. 
+
+Updates to an existing certified connector can be made through a simple pull request to the ```dev``` branch to update the certified connector files. 
+This will be reviewed by the connector owner. Once a PR has been merged to the ```dev``` branch, the connector owner can submit the connector for certification through the Connector certification tab in [ISV Studio](https://isvstudio.powerapps.com).
```
Comment (dougelliott) [inline README.md:84]: These two sections appear to have a somewhat similar message but are phrased differently. Are we addressing two separate audiences here, the connector owner and a third party member of the community looking to contribute? It's not immediately apparent to me why there are two distinct sections here.

Comment (bezhan-msft) [reply README.md]: That was my glossy mind writing this out too late at night - good catch and resolved.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,109 @@
...
+
+```origin  https://github.com/YOUR_USERNAME/PowerPlatformConnectors.git``` (push)
+
+Add an upstream repository for your clone.
+
+```git remote add upstream https://github.com/microsoft/PowerPlatformConnectors.git```
+
+Verify the upstream links.
+
+```git remote -v```
```
Comment (mamurshe) [inline README.md:42]: This is command.

File: README.md [hunk 3]
```diff
@@ -1,34 +1,109 @@
...
+
+```origin    https://github.com/YOUR_USERNAME/PowerPlatformConnectors.git``` (fetch)
```
Comment (mamurshe) [inline README.md:44]: These are outputs. Each line are in separate blocks. Doesn't look they are related.

**Comment by bezhan-msft** [comment]
Do not merge without engineering ack.

**Pull request merged**